src/keybindings.c

```diff
@@ -1848,10 +1845,14 @@ static gboolean cb_func_clipboard_action(guint key_id)
 			on_paste1_activate(NULL, NULL);
 			break;
 		case GEANY_KEYS_CLIPBOARD_COPYLINE:
-			sci_send_command(doc->editor->sci, SCI_LINECOPY);
+			if (!IS_SCINTILLA(focusw)) 
+				break;
+			sci_send_command(SCINTILLA(focusw), SCI_LINECOPY);
```

Comment:
style: I'd prefer to see `if (IS_SCINTILLA(...)) sci_send_command()` than `if (! IS_SCINTILLA(...)) break; sci_send_command()`.